### PR TITLE
Update API version.

### PR DESCRIPTION
**Pull request by kegor**
**Solution:**
- Were updated versions of the Conversion API and all other libraries.
- Was updated logic of the main `Convert` method.
- Changes were tested alongside with improvements from https://github.com/groupdocs-total/GroupDocs.Total-Angular/pull/172.

### CODE REVIEW

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Conversion/Manager/ConversionManager.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801&bid=17454486)

File: src/Products/Conversion/Manager/ConversionManager.cs [hunk 1]
```diff
@@ -1,57 +1,72 @@
...
                 }
-            }
-            else
-            {
-                convertedDocument.Save(resultFileName);
             }
         }
 
-        private SaveOptions GetSaveOptions(string sourceType, string destinationType)
+        private ConvertOptions GetConvertOptions(string destDocumentType, string destinationType)
```
Comment (bobkovalex) [inline src/Products/Conversion/Manager/ConversionManager.cs:41]: Codacy Issue found: [Make 'GetConvertOptions' a static method.](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Conversion/Manager/ConversionManager.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801&bid=17454486)

File: src/Products/Conversion/Manager/ConversionManager.cs [hunk 2]
```diff
@@ -1,57 +1,75 @@
...
```
Comment (bobkovalex) [inline src/Products/Conversion/Manager/ConversionManager.cs:41]: Codacy Issue found: [Make 'GetConvertOptions' a static method.](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801)

**Review by Pavel-Teplitsky — APPROVED** [review]

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Conversion/Controllers/ConversionApiController.cs  1
- src/Products/Conversion/Manager/ConversionManager.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801&bid=17454486)

File: src/Products/Conversion/Manager/ConversionManager.cs (hunk 2 above)
Comment (bobkovalex) [inline src/Products/Conversion/Manager/ConversionManager.cs:41]: Codacy Issue found: [Make 'GetConvertOptions' a static method.](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Conversion/Controllers/ConversionApiController.cs  1
- src/Products/Conversion/Manager/ConversionManager.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801&bid=17454486)

File: src/Products/Conversion/Manager/ConversionManager.cs [hunk 3]
```diff
@@ -1,57 +1,82 @@
...
```
Comment (bobkovalex) [inline src/Products/Conversion/Manager/ConversionManager.cs:48]: Codacy Issue found: [Make 'GetConvertOptions' a static method.](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/Products/Conversion/Controllers/ConversionApiController.cs  1
- src/Products/Conversion/Manager/ConversionManager.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-conversion/GroupDocs.Conversion-for-.NET-MVC/pullRequest?prid=5332801&bid=17454486)

**Pull request merged**